struct.rkt

```diff
@@ -129,6 +129,20 @@
                             #:trim-final-newlines (hash-ref jsexpr 'trimFinalNewlines undef-object)
                             #:key (hash-ref jsexpr 'key undef-object))))
 
+(struct SemanticToken
+  (start end type modifiers)
+  #:transparent)
+
+(define *semantic-token-types*
+  '(variable
+     function
+     string
+     number
+     regexp))
```

Comment:
Is this list order irrelevant?

Reply:
The list order is irrelevant.

The client get this list (in this order) from server ability during initialize handshake. Then it uses this list to decode the server messages.

The different order of this list produces different result of encoding. But it still works without change other code.

I should mention this in comment.